src/api/clients/scoped.ts

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import APIClientBase from "./base";
+import CONFIG from "@/config";
+import { APIError, APIRequestFailed } from "../index";
+import { getCookieFromHeader } from "@/api/helpers";
+
+export default class ScopedAPIClient extends APIClientBase {
+  constructor(cookieHeader?: string) {
+    
+    const csrfAccess = getCookieFromHeader(cookieHeader, "csrf_access_token");
```

Comment:
```suggestion
    const csrfAccess = getCookie(cookies, "csrf_access_token");
```
Rename cause they are not headers (just to don't mix concepts)